README.org

```diff
@@ -1,6 +1,45 @@
-This isn't anything yet. Go away.
+#+TITLE: README
+#+AUTHOR: Andrew Stahlman
```

Comment:
How about making the title "magit-gh-comments.el"? README seems kind of redundant.